content/en/_redirects

```diff
@@ -21,6 +21,33 @@
+/docs/components/pipelines/v2/installation/quickstart/                       /docs/components/pipelines/overview/introduction/
+/docs/components/pipelines/v2/compile-a-pipeline/                            /docs/components/pipelines/how-to/compile-a-pipeline/
+/docs/components/pipelines/v2/local-execution/                               /docs/components/pipelines/how-to/execute-kfp-pipelines-locally/
+/docs/components/pipelines/v2/cli/                                           /docs/components/pipelines/how-to/cli/
+/docs/components/pipelines/v2/components/lightweight-python-components/      /docs/components/pipelines/how-to/create-components/lightweight-python-components/
+/docs/components/pipelines/v2/pipelines/pipeline-basics/                     /docs/components/pipelines/how-to/create-components/compose-components-into-pipelines/
+/docs/components/pipelines/v2/components/containerized-python-components/    /docs/components/pipelines/how-to/create-components/containerized-python-components/
+/docs/components/pipelines/v2/components/container-components/               /docs/components/pipelines/how-to/create-components/container-components/
+/docs/components/pipelines/v2/components/importer-component/                 /docs/components/pipelines/how-to/create-components/importer-component/
+/docs/components/pipelines/v2/components/additional-functionality/           /docs/components/pipelines/how-to/create-components/additional-functionality/
+/docs/components/pipelines/v2/load-and-share-components/                     /docs/components/pipelines/how-to/load-and-share-components/
+/docs/components/pipelines/v2/pipelines/control-flow/                        /docs/components/pipelines/how-to/control-flow/
+/docs/components/pipelines/v2/data-types/parameters/                         /docs/components/pipelines/how-to/parameters/
+/docs/components/pipelines/v2/data-types/artifacts/                          /docs/components/pipelines/how-to/artifacts/
+/docs/components/pipelines/v2/caching/                                       /docs/components/pipelines/how-to/caching/
+/docs/components/pipelines/v2/administration/server-config/                  /docs/components/pipelines/how-to/server-config/
+/docs/components/pipelines/v2/migration/                                     /docs/components/pipelines/how-to/migration/
+/docs/components/pipelines/v2/platform-specific-features/                    /docs/components/pipelines/how-to/platform-specific-features/
+/docs/components/pipelines/v2/community-and-support/                         /docs/components/pipelines/reference/community-and-support/
+/docs/components/pipelines/v2/version-compatibility/                         /docs/components/pipelines/reference/version-compatibility/
+/docs/components/pipelines/v2/reference/api/kubeflow-pipeline-api-spec/      /docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/
+
```

Comment:
@hbelmiro  I recommend adding these to the end of the file (rather than the beginning), so they apply after any previous redirects, to prevent 404s.

It's fine to chain redirects.